src/api/v1/paths/doughs/{doughId}.js

```diff
@@ -0,0 +1,28 @@
+import { getDoughById } from 'api/v1/db/oracledb/doughs-dao';
+import { errorBuilder, errorHandler } from 'errors/errors';
+
+/**
+ * Get a dough with id `req.params.doughId`
+ *
+ * @type {RequestHandler}
+ */
+const get = async (req, res) => {
+  try {
+    const result = await getDoughById(req.params.doughId);
+    if (result.data.length === 0) {
+      return errorBuilder(res, '404', `No dough with id ${req.params.doughId} exists`);
```

Comment:
not sure if there's a standard text for this detail message, or if it's actually necessary

Reply:
That looks fine to me, the only comment I have is you probably wanna change `id` to `ID` in text.